release/models/bgp/openconfig-bgp-types.yang

```diff
@@ -795,10 +801,8 @@ module openconfig-bgp-types {
       }
       enum NONE {
         description
-            "Do not send any community attribute.
-            This value has been deprecated because the node is now
-            a leaf-list.";
-        status deprecated;
+            "Do not send any community attribute. If this value is present
+            then no communities will be sent.";
```

Comment:
This value must not coexist with any other value in same community type list.
[NONE STANDARD] is ambiguous.

Reply:
Done